doc/source/data/batch_inference.rst

```diff
@@ -0,0 +1,752 @@
+.. _batch_inference_home:
+
+Running Batch Inference with Ray
+================================
+
+.. seealso::
+
+    In this tutorial you'll learn what batch inference is, why you might want to use
+    Ray for it, and how to use Ray effectively for this task.
+    If you are familiar with the basics of inference tasks, you can jump straight into
```

Comment:
```suggestion
    If you are familiar with the basics of inference tasks, jump straight to
```

Reply:
Suggest not starting off the doc with a `See also` callout and rely on the TOC on the RHS. I noticed that the RHS TOC is collapsed by default on this page, where it's expanded by default on other pages.

Reply:
no, it's hidden when there's stuff on the right, like the margin I had there (should be gone now)